packages/app-data/sheets/template/debug/debug_nesting_conditional.json

```diff
@@ -165,7 +165,10 @@
           "action_id": "set_local",
           "args": [],
           "_raw": "click | set_local | var_toggle: !@local.var_toggle",
-          "_cleaned": "click | set_local | var_toggle: !@local.var_toggle"
```

Comment:
Will syntax 
`changed | set_field: @local.slider_tracker: @local.slider; changed | set_local : slider_answered : true` still work or should this be refactored everywhere to be `changed | set_field | @local.slider_tracker: @local.slider; changed | set_local | slider_answered : true`?

Reply:
I've just copied the above and added line breaks to try and see the difference. 
```
changed | set_field: @local.slider_tracker: @local.slider; 
changed | set_local : slider_answered : true
```
```
changed | set_field | @local.slider_tracker: @local.slider; 
changed | set_local | slider_answered : true
```
I wasn't aware we are using two different syntaxes for authoring set_local, but seeing as it has only been picked up in the one example template I'm assuming this is actually a typo (?). In either case this shouldn't change the handling of existing set_field and set_local operations, but does open the door if we want to add support for providing a list of values to change e.g.
```
changed | set_field | field_a:value_a, field_b:value_b, field_c: value_c;
```
This would still require some additional code changes to the set_field and set_local actions (and anywhere else we wanted to provide), but could be useful to have. In this case we would aim to keep both options available at least in the short term

Reply:
(I just checked the file history and indeed it looks like this was just an authoring typo)